### PR TITLE
Update downtime-notifications-uptimerobot.md

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

File: content/sre/alerting/downtime-notifications-uptimerobot.md [hunk 1]
```diff
@@ -12,5 +19,26 @@ Stakater App Agility Platform provides downtime notifications for Applications v
 - After picking a channel or user to be notified, click the `Add Incoming WebHooks Integration` Button. The most important part on the next screen is the `WebHook URL`. Make sure you copy this URL and send it to Stakater Support
 - Near the bottom of this page, you may further customize the Incoming WebHook you just created. Give it a name, description and perhaps a custom icon.
 
-## Items to be provided to Stakater Support
+### Items to be provided to Stakater Support
 - `Incoming WebHook URL`
+
+## 2. Create alert contact on uptimerobot with webhook 
+
+Create alert contact on uptimerobot
```
Comment (hanzala1234) [inline content/sre/alerting/downtime-notifications-uptimerobot.md:27]: No step for this?

File: content/sre/alerting/downtime-notifications-uptimerobot.md [hunk 2]
```diff
@@ -12,5 +19,26 @@ Stakater App Agility Platform provides downtime notifications for Applications v
...
+
+## 3. Update IMC configuration
+
+Update IngressMonitorController configuration
```
Comment (hanzala1234) [inline content/sre/alerting/downtime-notifications-uptimerobot.md:31]: same for this as well?

File: content/sre/alerting/downtime-notifications-uptimerobot.md (hunk 1 above)
Comment (rasheedamir) [reply content/sre/alerting/downtime-notifications-uptimerobot.md:27]: will be added later with screenshots; right now it has been taken care of by genesis team already

File: content/sre/alerting/downtime-notifications-uptimerobot.md (hunk 2 above)
Comment (rasheedamir) [reply content/sre/alerting/downtime-notifications-uptimerobot.md:31]: will be added later with screenshots; right now it has been taken care of by genesis team already

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-185-e0f17536`

**Review by hanzala1234 — APPROVED** [review]

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.126`

**Pull request merged**